spec/02_state.md

```
@@ -11,10 +11,12 @@ Vaults may be configured with an optional **payment denom** in addition to the *
   - [Vault Lookup (prefix 0)](#vault-lookup-prefix-0)
   - [Payout Verification Set (prefix 1)](#payout-verification-set-prefix-1)
   - [Payout Timeout Queue (prefix 2)](#payout-timeout-queue-prefix-2)
+  - [Vault Fee Timeout Queue (prefix 7)](#vault-fee-timeout-queue-prefix-7)
   - [Pending Swap-Out Queue (prefix 3)](#pending-swapout-queue-prefix-3)
```

markdownlint-cli2 (0.22.0)

[warning] Link fragments should be valid

(MD051, link-fragments)